org.compiere.db.oracle.provider/META-INF/pool/server.default.properties

```diff
@@ -1,18 +1,76 @@
-#timeout
-IdleConnectionTestPeriod=1200
-AcquireRetryAttempts=2
-MaxIdleTimeExcessConnections=1200
-MaxIdleTime=1200
-#UnreturnedConnectionTimeout=1800
-
-#size
-MaxPoolSize=150
-InitialPoolSize=10
-MinPoolSize=5
-MaxStatementsPerConnection=30
-
-#flag
-TestConnectionOnCheckin=false
-TestConnectionOnCheckout=true
-#CheckoutTimeout=60;
-com.mchange.v2.log.MLog=com.mchange.v2.log.slf4j.Slf4jMLog
\ No newline at end of file
+# !! ALL SETTINGS PRESENT IN THIS FILE WILL BE FED IN TO HIKARICP !!
+#                  !! DO NOT SET EMPTY VALUES !!
+#  
+# You can add HikariCP settings that are not present in this file. In order to
+# use the default just remove or comment out the key all together.
+
+cachePrepStmts=true
+prepStmtCacheSize=250
+prepStmtCacheSqlLimit=2048
```

Comment:
isn't Hikari CP recommend against the use of prepare statement cache ?

Reply:
I assume you are referring to [this section](https://github.com/brettwooldridge/HikariCP/blob/ef99317811093299858d27b54be7b40c43224901/README.md#statement-cache) in HikariCP's readme?

The way I read it is that HikariCP doesn't have a prepared statement cache (because they indeed don't recommend it). However they do recommend to use the JDBC / DB's native statement cache. If I understand it correctly the `prepStmtCacheSqlLimit` is a setting used by the datasource's JDBC driver.

Though something is still wrong here since I think these settings are MySQL specific and don't do anything for PostgreSQL. I'll remove them.